jdk-postgres/8-15/Dockerfile

```diff
@@ -0,0 +1,221 @@
+#
+# NOTE: This dockerfile is based on the official postgres dockerfile
+# See https://github.com/docker-library/postgres
+#
+
+FROM ghcr.io/scalar-labs/jdk8:1.0.0
```

Comment:
This assumes that `jdk8` will be released with the tag of `1.0.0`

Reply:
Addressed by https://github.com/scalar-labs/docker/pull/20/commits/9b2c63cffbb8ebb0c2da2c9f14a13de2090b6f8d